core/include/crypto/crypto_impl.h

```diff
@@ -297,4 +298,86 @@ drvcrypt_mac_alloc_ctx(struct crypto_mac_ctx **ctx __unused,
 	return TEE_ERROR_NOT_IMPLEMENTED;
 }
 #endif /* CFG_CRYPTO_DRV_MAC */
+
+/*
+ * The ECC public key operations used by the crypto_acipher_ecc_*() and
+ * crypto_acipher_free_ecc_*() functions.
+ * Reference set in ecc_public_key when key allocated.
```

Comment:
Please mention which function pointers are optional and which are mandatory. Same for `struct crypto_ecc_keypair_ops` below.

Reply:
OK